index.html

```diff
@@ -274,6 +275,43 @@ <h2>Network requests</h2>
       <a>HTTP-network fetch</a> algorithm.
       </p>
 
+      <p>
+      Regardless of which fetch algorithm and which underlying application and
+      transport protocols are used, servicing a <a>network request</a> consists
+      of the following <dfn data-lt="phase">phases</dfn>:

```

Comment:
I'd love if these phases were defined in a more central place, like Fetch. That'd make it more likely that specs like the [Web Packaging specs](https://github.com/WICG/webpackage) remember to define which phase they're operating in.

Reply:
I've added an editor's note that we'd like to move these into Fetch.  I'd like to keep the definitions here so that they're available somewhere until that happens.